articles/azure/azs-how-resize-disks.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to resize virtual machine disks
+description: Learn how to resize virtual machine disks
+services: azure-stack
+author: David Woffendin 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Disk Management
+toc_sub3:
+toc_sub4:
+toc_title: Resize virtual machine disks
+toc_fullpath: Users/How To/Disk Management/azs-how-resize-disks.md
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks
+
+In this article we will cover the process of resizing disks in the Azure portal. If your VM requires more storage space, you can increase the current disk capacity, rather than adding more disks. You can do this in the Azure portal.
+
+## Resizing disk size via the portal
+
+These steps will work for both non managed and managed VM's.
```

Comment:
Should be unmanaged instead on non managed

Reply:
done